docs/secret_management.md

```diff
@@ -43,113 +101,84 @@ spec:
         keyName: app-key
   # ...
 ```
+## Use the secret backend
 
-Create the Secret(s) before applying the DatadogAgent manifest, or the deployment will fail.
-
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  name: test-api-secret
-data:
-  api_key: <api-key>
-
----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: test-app-secret
-data:
-  app_key: <app-key>
-```
-
-**Note:**
+The Datatog Operator is compatible with the [secret backend][1] implemented.
```

Comment:
```suggestion
The Datatog Operator is compatible with the [secret backend][1] implemented.
```
this sentence is a little confusing, does it mean "compatible when the secret backend is implemented"?